### DISCUSSION TITLE
Thank you very much!

### DISCUSSION THREAD

**Discussion by stueja** [jonaswinkler/paperless-ng#528] [General]
Jonas, thank you very much for this wonderful piece of development. I had been using the original paperless software for like the last 3 years or so. I am using it daily to scan my daily papers, mail, contracts, insurances, bills, letters, ... everything.

Migration was a breeze thanks to your documentation, even for me, who has not yet quite understood docker-compose.

The paperless-ng user interface is SO fast and snappy! And I assume that the postgres container will allow simultaneous write action to the database so that I don't have to see django's error messages any more.

Wonderful. Simply wonderful. Thank you.

**Comment by jonaswinkler** [comment]
Thank you so much for that comment! :smiley:

I spend some major efforts on making sure that the migration works and tried to make the migration instructions as fail-proof as possible. I'm glad it worked out for you.

If you were referring to "Database is locked" errors, yes, those will be gone with postgres and *mostly* gone with SQLite (I've made a couple changes so that the database is only locked when absolutely necessary.

**Comment by stephanlinke87** [comment]
Little late to the party, but thanks from me as well @jonaswinkler and other contributors :) I always wanted to get into document management, but never found the right piece of software. Paperless-NG was the final straw that got me hooked. Great work, I really appreciate it :)

**Comment by Conspir3D** [comment]
Me too ... just discovered paperless-ng it it serves everything it needs for an easy every-day-dms for small companies or private people.
But the lack of some essential search stuff like and/or to tags etc. makes it sometimes a little bit unusable but with the FILENAME_FORMAT i can handle it on fs.
Thanks for this wonderfull piece of code which runs very very stable on my bare metal install

**Reply by Conspir3D** [reply]
ah and yes ... mysql support would be great ;)

**Reply by Conspir3D** [reply]
nevermind my "search" comment ... just stumbled across the doc for that ;)

**Comment by frank-from-cologne** [comment]
Also big THANK YOU from me

Danke sehr aus Köln

**Comment by torbengb** [comment]
Und aus Wien ❤
